includes/admin/list-tables/class-wc-admin-list-table-products.php

```diff
@@ -520,18 +520,16 @@ public function posts_clauses( $args, $query ) {
 	/**
 	 * Remove ordering queries.
 	 *
-	 * @param array $posts Posts from WP Query.
-	 * @return array
+	 * @return void
 	 */
-	public function remove_ordering_args( $posts ) {
+	public function remove_ordering_args() {
```

Comment:
We probably should keep `$posts` since it's a public method, also people may extend this class.
Probably `$posts = array()` should solve this problem, also keep returning `$posts`.